### PR TITLE
Sync VolumeHealthMonitor variable name with other platforms

### PR DESCRIPTION
**Pull request by panigs7**
# Description
This PR addresses the following:
1. Fixes dependabot alert
2. Change VolumeHealthMonitor variable name to healthMonitor to sync with other platforms

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/203|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Tested VolumeHealthMonitor feature by installing driver.
Controller pod logs:
controller status
Node pod logs:
node log

### CODE REVIEW

**Review by shanmydell — APPROVED** [review]

**Comment by shanmydell** [comment]
@panigs7 : Have you checked the variable in the operator code? Please make changes in documentation as well

**Comment by prablr79** [comment]
@panigs7 is this updated in Documentation as well ?

**Comment by panigs7** [comment]
> @panigs7 : Have you checked the variable in the operator code? Please make changes in documentation as well

I will raise separate PRs for operator changes and documentation.

**Review by rensyct — DISMISSED** [review]

**Review by rajendraindukuri — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/dell/dell-csi-extensions/podmon v1.0.0
 	github.com/dell/gobrick v1.2.0
 	github.com/dell/gocsi v1.5.0
-	github.com/dell/gofsutil v1.6.0
+	github.com/dell/gofsutil v1.7.1-0.20220204052137-9928a2dc48d8
```
Comment (karthikk92) [inline go.mod:11]: can we put this line inside replace , eg: like how we added for gounity : replace (
	github.com/dell/gounity => github.com/dell/gounity v1.8.1-0.20220117113341-b5ac52a19020)

**Review by karthikk92 — APPROVED** [review]

**Review by rajendraindukuri — APPROVED** [review]

**Pull request merged**